packages/vue3/src/components/Modal/Modal.vue

```diff
@@ -79,7 +93,7 @@ const closeModal = () => {
   }
 
   .header {
-    padding: 14px 20px;
+    padding: 14px 4px 14px 20px;
```

Comment:
do we have a right padding of `4px` ? Perhaps we can unify the padding to:
```suggestion
    padding: 14px 20px;
```

Reply:
Same thing for the `.footer`